docs/source/conf.py

```diff
@@ -97,12 +97,13 @@
 # a list of builtin themes.
 #
 html_theme = "sphinx_rtd_theme"
+html_logo = str((root_folder / "docs" / "source" / "images" / "logo.png").resolve())
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
 #
-# html_theme_options = {}
+html_theme_options = {"logo_only": True, "display_version": True}
```

Comment:
Where is this used?

Reply:
This will display the logo in the top-left corner of the docs and also display the package version below it